samples/boards/nrf/clock_skew/README.rst

```diff
@@ -8,6 +8,8 @@ Overview
 
 This sample uses the API for correlating time sources to measure the
 skew between HFCLK (used for the CPU) and LFCLK (used for system time).
+It also shows how to use this slew to correct durations measured in the
```

Comment:
Slew, or skew?

Reply:
Probably skew, since that's what I used everywhere else.  Fixed, thanks.